modules/oc-mirror-oci-format.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-oci-format_{context}"]
+= Mirroring images in OCI format
```

Comment:
@sherine-k @lmzuccarelli Do you think I should title this "Mirroring Operator images in OCI format" since this is only for Operators?

Reply:
@bergerhoffer - Yes I would agree, we don't want to confuse the issue. We don't offer "release" , "additionlaImages" and "helm" options - so for me this is to the point

Reply:
Thanks, updated